CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * [CHANGE] Restructure Azure backends into versioned backends.  Introduce `use_v2_sdk` config option for switching. [#2952](https://github.com/grafana/tempo/issues/2952) (@zalegrala)
     v1: [azure-storage-blob-go](https://github.com/Azure/azure-storage-blob-go) original (now deprecated) SDK
     v2: [azure-sdk-for-go](https://github.com/Azure/azure-sdk-for-go)
+* [CHANGE] Adjust trace size estimation to better honor row group size settings. [#2952](https://github.com/grafana/tempo/pull/3038) (@joe-elliott)
```

Comment:
@joe-elliott Heads-up, text and link mismatch

Reply:
Nice catch! Thx